lib/float.gd

```diff
@@ -44,87 +45,269 @@ end);
 ##
 #O Unary operations
 ##
-DeclareAttribute("Cos",IsFloat);
-DeclareAttribute("Sin",IsFloat);
-DeclareAttribute("Tan",IsFloat);
-DeclareAttribute("Sec",IsFloat);
-DeclareAttribute("Csc",IsFloat);
-DeclareAttribute("Cot",IsFloat);
-DeclareAttribute("Asin",IsFloat);
-DeclareAttribute("Acos",IsFloat);
-DeclareAttribute("Atan",IsFloat);
-DeclareAttribute("Cosh",IsFloat);
-DeclareAttribute("Sinh",IsFloat);
-DeclareAttribute("Tanh",IsFloat);
-DeclareAttribute("Sech",IsFloat);
-DeclareAttribute("Csch",IsFloat);
-DeclareAttribute("Coth",IsFloat);
-DeclareAttribute("Asinh",IsFloat);
-DeclareAttribute("Acosh",IsFloat);
-DeclareAttribute("Atanh",IsFloat);
-DeclareOperation("Log",[IsFloat]);
-DeclareAttribute("Log2",IsFloat);
-DeclareAttribute("Log10",IsFloat);
-DeclareAttribute("Log1p",IsFloat);
-DeclareAttribute("Exp",IsFloat);
-DeclareAttribute("Exp2",IsFloat);
-DeclareAttribute("Exp10",IsFloat);
-DeclareAttribute("Expm1",IsFloat);
-DeclareAttribute("CubeRoot",IsFloat);
-DeclareAttribute("Square",IsFloat);
-DeclareAttribute("Ceil",IsFloat);
-DeclareAttribute("Floor",IsFloat);
-DeclareAttribute("Round",IsFloat);
-DeclareAttribute("Trunc",IsFloat);
-DeclareOperation("Atan2", [IsFloat,IsFloat]);
+## <#GAPDoc Label="Float-Math-Commands">
+## <ManSection>
+##   <Heading>Standard mathematical operations</Heading>
+##   <Oper Name="Cos" Arg="f"/>
+##   <Oper Name="Sin" Arg="f"/>
+##   <Oper Name="Tan" Arg="f"/>
+##   <Oper Name="Sec" Arg="f"/>
+##   <Oper Name="Csc" Arg="f"/>
+##   <Oper Name="Cot" Arg="f"/>
+##   <Oper Name="Asin" Arg="f"/>
+##   <Oper Name="Acos" Arg="f"/>
+##   <Oper Name="Atan" Arg="f"/>
+##   <Oper Name="Cosh" Arg="f"/>
+##   <Oper Name="Sinh" Arg="f"/>
+##   <Oper Name="Tanh" Arg="f"/>
+##   <Oper Name="Sech" Arg="f"/>
+##   <Oper Name="Csch" Arg="f"/>
+##   <Oper Name="Coth" Arg="f"/>
+##   <Oper Name="Asinh" Arg="f"/>
+##   <Oper Name="Acosh" Arg="f"/>
+##   <Oper Name="Atanh" Arg="f"/>
+##   <Oper Name="Log" Arg="f"/>
+##   <Oper Name="Log2" Arg="f"/>
+##   <Oper Name="Log10" Arg="f"/>
+##   <Oper Name="Log1p" Arg="f"/>
+##   <Oper Name="Exp" Arg="f"/>
+##   <Oper Name="Exp2" Arg="f"/>
+##   <Oper Name="Exp10" Arg="f"/>
+##   <Oper Name="Expm1" Arg="f"/>
+##   <Oper Name="CubeRoot" Arg="f"/>
+##   <Oper Name="Square" Arg="f"/>
+##   <Oper Name="Ceil" Arg="f"/>
+##   <Oper Name="Floor" Arg="f"/>
+##   <Oper Name="Round" Arg="f"/>
+##   <Oper Name="Trunc" Arg="f"/>
+##   <Oper Name="Atan2" Arg="y x"/>
+##   <Oper Name="FrExp" Arg="f"/>
+##   <Oper Name="LdExp" Arg="f exp"/>
+##   <Oper Name="AbsoluteValue" Arg="f"/>
+##   <Oper Name="Norm" Arg="f"/>
+##   <Oper Name="Hypothenuse" Arg="x y"/>
+##   <Oper Name="Frac" Arg="f"/>
+##   <Oper Name="SinCos" Arg="f"/>
+##   <Oper Name="Erf" Arg="f"/>
+##   <Oper Name="Zeta" Arg="f"/>
+##   <Oper Name="Gamma" Arg="f"/>
+##   <Description>
+##     These are all standard math functions, directly copied from the standard math library; see <C>man 3 math</C> for their list.
```

Comment:
When you say "standard math library", what you mean? The C math library? Or maybe the POSIX math library? Or the linux (glibc) math library? Or something else?

These are all different things. And for me (on OS X), several of the functions listed above have either a different name in `man 3 math` (ignore case differences), e.g. CubeRoot` (vs `cbrt`), `Square` (vs. `sqrt`), `Hypothenuse` (vs. `hypot`), or they are missing altogether, e.g. `Zeta` , `Frac`, `SinCos`, `Gamma`

So I would be very hesitant to mention `man 3 math` here, at least like this. I really think there should be one mansection for each of the above, with a description of what it does, and possibly the name of the corresponding function in a specific standard, such as ISO C, and/or perhaps references to Clause 9  of IEEE 754.